test/gw-ci/create_exp.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+expyaml="$1"
+export pslot="$2"
+HOMEgfs="$3"
+export RUNTESTS="$4"/${pslot}
+export ICSDIR_ROOT=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
```

Comment:
This is Hera specific, can we support other platforms too?

Reply:
arf ... will add the other machines